lib/systems/include/PedalsSystem.h

```diff
@@ -109,42 +98,22 @@ class PedalsSystem
     /// @param params
     /// @param max_percent_diff
     /// @return
-    bool evaluate_pedal_implausibilities_(int pedalData1_analog,
-                                          int pedalData2_analog,
+    bool _evaluate_pedal_implausibilities(int pedal_1_raw,
+                                          int pedal_2_raw,
                                           const PedalsParams &params,
                                           float max_percent_diff);
 
-    /// @brief overloaded pedal implaus check that doesnt need to check for percent diff between sensors since only one sensor
-    /// @param pedalData
-    /// @param params
-    /// @return
-    bool evaluate_pedal_implausibilities_(int pedalData, const PedalsParams &params);
-
     /// @brief function to determine if the pedals and the brakes are pressed at the same time.
     ///        evaluates brake being pressed with mech brake activation threshold AFTER removing
     ///        deadzones for both brake and accel
-    /// @param accelPedalData1
-    /// @param accelPedalData2
-    /// @param brakePedalData1
-    /// @param brakePedalData2
-    /// @return true if accel and brake pressed at the same time, false otherwise
-    bool evaluate_brake_and_accel_pressed_(int accelPedalData1_analog,
-                                           int accelPedalData2_analog,
-                                           int brakePedalData1_analog,
-                                           int brakePedalData2_analog);
-
-    /// @brief overloaded version that evaluates with only one brake pedal value
-    /// @param accelPedalData1
-    /// @param accelPedalData2
-    /// @param brakePedalData
-    /// @return
-    bool evaluate_brake_and_accel_pressed_(int accelPedalData1_analog,
-                                           int accelPedalData2_analog,
-                                           int brakePedalData_analog);
+    /// @param pedal_data
+    /// @param twopedals
+    bool _evaluate_brake_and_accel_pressed(PedalSensorData_s & pedal_data);
+
     /// @brief This checks to see if any pedal sensor is out of range :(
     /// @param PedalData The analog pedal Value
```

Comment:
This param does not match up with the function it's documenting. Also, is "out of range" the _sensor_ out of range, or is it out of the calibrated range? (i.e. if the sensor can return values 90-4000, but the calibrated range is 1000-3000, then is 500 OOR?) Please specify in documentation